skills/README.md

```
@@ -0,0 +1,70 @@
+
+## Installing
+
+Any agent (Claude Code, Cursor, Codex, Copilot, opencode, Zed, and around seventy others):
+
+```bash
+npx skills add ls1intum/Artemis
+```
+
+Claude Code, as a versioned plugin with namespaced skills (`/artemis:e2e-pr-check`):
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)